contracts/utils/math/SafeMath.sol

```diff
@@ -28,7 +28,7 @@ library SafeMath {
     }
 
     /**
-     * @dev Returns the substraction of two unsigned integers, with an overflow flag.
+     * @dev Returns the subtraction of two unsigned integers, with an overflow flag.
```

Comment:
I don't think that change is correct

Reply:
![image](https://user-images.githubusercontent.com/45835846/159381146-eb1c50f7-79f7-4e00-a5f3-f2a5917be4aa.png)

Reply:
And Wikipedia also agrees...

Note that Google translate caused the confusion in the first place as it says "substraction" when translating "soustraction" from the French.